### PR TITLE
Update routing-with-expressroute.md

### PR DESCRIPTION
**Pull request by troofeds**

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/routing-with-expressroute.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Some of the key items in the above articles that you'll need to understand inclu
 
 - There's a 1:1 mapping between an ExpressRoute circuit and a customer s-key.
 
-- Each circuit can support up to 3 independent peering relationships (Azure Public peering, Azure Private peering, and Microsoft peering); Office 365 requires Microsoft peering.
+- Each circuit can support 2 independent peering relationships (Azure Private peering, and Microsoft peering); Office 365 requires Microsoft peering.
```
Comment (nenonix) [inline Enterprise/routing-with-expressroute.md]: Each circuit can support **two** independent peering relationships (Azure **p**rivate peering, and Microsoft peering); Office 365 requires Microsoft peering.

Comment (JohanFreelancer9) [reply Enterprise/routing-with-expressroute.md]: Agree.

**Comment by mypil** [comment]
@kelleyvice-msft - Can you please assist to check, approve and merge the content of this PR?

Thank you.

**Comment by mypil** [comment]
@kelleyvice-msft - We would like to follow up on this. Can you please assist to check, approve and merge the content of this PR?

Thank you.

**Comment by mypil** [comment]
@kelleyvice-msft - We would like to follow up on this. Can you please assist to check, approve and merge the content of this PR? Thank you.

cc: @LauraWi

**Comment by kelleyvice-msft** [comment]
@mypil - sorry! The notifications were going into an Outlook subfolder. Can you tell me where you got verification of the change? If it's been verified, I will merge the PR.

**Comment by mypil** [comment]
@kelleyvice-msft - this was a PR created by @troofeds, there is no issue related to this.

@troofeds - can you tell us where you got verification of the change?

**Comment by mypil** [comment]
@troofeds -We would like to follow up.  Can you please tell us where you got verification of the change?

**Comment by troofeds** [comment]
https://docs.microsoft.com/en-gb/azure/expressroute/expressroute-introduction

**Comment by mypil** [comment]
@kelleyvice-msft - kindly refer to the reply above from @troofeds. Please check if this would suffice to merge this PR. thank you.

**Comment by kelleyvice-msft** [comment]
Looks good. Thanks for the update!

**Pull request merged**